v2.0/paths/task_assignments__get_task_assignments_id.yml

```diff
@@ -22,3 +22,6 @@ responses:
       application/json:
         schema:
           $ref: '#/components/schemas/TaskAssignment'
+  404:
```

Comment:
Does this response also include a body? 

Can we make the description match the format “Returns an error if…” as per https://github.com/box/box-openapi/blob/master/v2.0/paths/chunked_uploads__post_files_id_upload_sessions.yml#L51?